components/gallery/Gallery.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div>
+        <button v-on:click="prevImg">prev</button>
```

Comment:
давай будем проще, v-on писать не обязательно,
а еще можно избавиться от необходимости писать `event.preventDefault()` если прямо здесь использовать [модификатор события](https://ru.vuejs.org/v2/guide/events.html#%D0%9E%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%B8-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9)
```suggestion
        <button @click.prevent="prevImg">prev</button>
```